api/v1alpha1/managedocs_types.go

```diff
@@ -35,7 +35,8 @@ const (
 
 // ManagedOCSSpec defines the desired state of ManagedOCS
 type ManagedOCSSpec struct {
-	ReconcileStrategy ReconcileStrategy `json:"reconcileStrategy,omitempty"`
+	ReconcileStrategy              ReconcileStrategy `json:"reconcileStrategy,omitempty"`
+	NetworkPolicyReconcileStrategy ReconcileStrategy `json:"networkPolicyReconcileStrategy,omitempty"`
```

Comment:
I think this will become very messy very fast. 
One option is to elevate the original `ReconcileStrategy` to a "master switch",  then have a new sub-spec struct containing specific reconcile switches, based on the different aspects that we would like the control on. 
For now, these will be ODFDeployment and NetworkPolicies

Reply:
Do you mean something like this?
```
...
type ManagedOCSSpec struct{
  MasterReconcileStrategy ReconcileStrategy
  ComponentReconcileStrategy ComponentReconcileStrategy
  }
  ----
  
  type ComponentReconcileStrategy struct{
   ODFDeployment ReconcileStrategy
   NetworkPolicies ReconcileStrategy
 }
 ```
  
  

 ```

Reply:
Something like that, but we cannot rename `reconcilePolicy` to `mainReconcilePolicy` as this is a breaking CR change